experiment_prototype/experiment_prototype.py

```diff
@@ -1608,9 +1608,6 @@ def check_slice(self, exp_slice):
             if not exp_slice['intt']:
                 error_list.append("Slice {} must have intt enabled to use scanbound".format(
                         exp_slice['slice_id']))
-            elif len(exp_slice['scanbound']) != len(exp_slice['beam_order']):
```

Comment:
Shouldn't these checks be kept, in case there are no INTTIME interfaced slices? 
Would it break if a slice contained a scanbound, but someone didn't program it correctly, for example having the number of beams equal 16, but the number of scanbound entries equal 15? 

Maybe I am missing the point of having a scanbound?